skimage/feature/_canny.py

```diff
@@ -50,7 +50,8 @@ def smooth_with_function_and_mask(image, function, mask):
     return output_image
 
 
-def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None):
+def canny(image, sigma=1., low_threshold=None, high_threshold=None, mask=None,
+    quantile_threshold=False):
```

Comment:
The name makes me think it is a floating point number or a tuple, perhaps `use_quantiles`?